code/modules/clothing/spacesuits/rig/suits/station.dm

```diff
@@ -63,6 +63,7 @@
 	desc = "A heavy, powerful rig used by construction crews and mining corporations."
 	icon_state = "industrial_rig"
 	armor = list(melee = 60, bullet = 40, laser = 30, energy = 15, bomb = 30, bio = 100, rad = 50)
+	siemens_coefficient = 0.1
```

Comment:
Seems a bit excessive for the industrial rig since thats not `Super-Heavy Armor`
-> Stick with the default for rigs